fastlane/lanes/release.rb

```diff
@@ -120,7 +125,7 @@
       end
     end
 
-    setfrozentag(repository: GITHUB_REPO, milestone: new_version)
+    set_milestone_frozen_marker(repository: GITHUB_REPO, milestone: new_version)
```

Comment:
After leaving my comment above, I realize I didn't update this one adding the token.

I guess it didn't occur to me at rename time because I did a mechanical find and replace.